play/src/front/Chat/Connection/Matrix/MatrixChatConnection.ts

```diff
@@ -157,8 +145,7 @@ export class MatrixChatConnection implements ChatConnectionInterface {
                 const newRoom = new MatrixChatRoom(room);
                 if (
                     inviter &&
-                    (this.userDisconnected.has(inviter) ||
-                        Array.from(get(this.connectedUsers).values()).some((user: ChatUser) => user.chatId === inviter))
+                    this.AllWorldUserspaceFilter.getUsers().find((user: SpaceUserExtended) => inviter === user.chatID)
```

Comment:
We should find a way NOT to use the AllWorldUserspaceFilter in the MatrixChatConnection.
Ideally, this should be only used in the user list (in a "readable" that makes it lazy loaded).
This way, we will not receive any network calls from changes in the world space unless we open the user list (this could save a lot of CPU and bandwidth for users with 1000 users)